Frameworks/UIKit/UIPanGestureRecognizer.mm

```diff
@@ -478,6 +478,7 @@ - (_UIPanGestureStage)_stage {
 */
 - (void)reset {
     _priv->touches.clear();
+	_priv->currentTranslation.x = _priv->currentTranslation.y = 0.f;
```

Comment:
@jaredhms The original code that sets the currentTranslation to 0 does a simultaneous assignment.  Should we change both at the same time to be consistent?